po/fr.po

```diff
@@ -0,0 +1,296 @@
+"SerenityOS project."
+msgstr ""
+"Ladybird est un navigateur basé sur le moteur de rendu LibWeb et la bibliothèque JavaScript "
+"LibJS, développé par une équipe de contributeurs appartenant au projet SerenityOS."
+
+#: window.blp:218
+msgid "Main Menu"
+msgstr "Menu Principal"
+
+#: window.blp:145 window.blp:282
+msgid "Navigate Back"
+msgstr "Naviguer vers la page précédente"
+
+#: window.blp:157 window.blp:288
+msgid "Navigate Forward"
+msgstr "Naviguer vers la page suivante"
+
+#: shortcuts-dialog.blp:38
+msgid "Navigate back"
+msgstr "Page précédente"
+
+#: shortcuts-dialog.blp:42
+msgid "Navigate forward"
+msgstr "Page suivante"
+
+#: shortcuts-dialog.blp:35
+msgid "Navigation"
+msgstr "Navigation"
+
+#: window.blp:175
+msgid "New Tab"
+msgstr "Nouvel Onglet"
+
+#: window.blp:11 window.blp:43
+msgid "New _Incognito Window"
+msgstr "Nouvelle Fenêtre de Navigation _Privée"
+
+#: shortcuts-dialog.blp:17
+msgid "New incognito window"
+msgstr "Nouvelle fenêtre privée"
+
+#: Window.cpp:101 shortcuts-dialog.blp:59
+msgid "New tab"
+msgstr "Nouvel onglet"
+
+#: shortcuts-dialog.blp:13
+msgid "New window"
+msgstr "Nouvelle fenêtre"
+
+#: shortcuts-dialog.blp:67
+msgid "Next tab"
+msgstr "Onglet suivant"
+
+#: Window.cpp:236 Tab.cpp:194
+msgid "OK"
+msgstr "OK"
+
+#: shortcuts-dialog.blp:21
+msgid "Open file"
+msgstr "Ouvrir un fichier"
+
+#. Translators: this is a menu item for pinning a tab.
+#: window.blp:26
+msgid "P_in"
+msgstr "_Épingler"
+
+#: shortcuts-dialog.blp:71
+msgid "Previous tab"
+msgstr "Onglet précédent"
+
+#: shortcuts-dialog.blp:29
+msgid "Quit"
+msgstr "Quitter"
+
+#: window.blp:169 window.blp:294
+msgid "Reload Page"
+msgstr "Recharger la Page"
+
+#: shortcuts-dialog.blp:46
+msgid "Reload current page"
+msgstr "Actualiser"
+
+#: shortcuts-dialog.blp:89
+msgid "Reset zoom"
+msgstr "Taille réelle"
+
+#: window.blp:236
+msgid "Restore Zoom"
+msgstr "Taille réelle"
+
+#: Application.cpp:49
+msgid "Sam Atkins"
+msgstr "Sam Atkins"
+
+#. Translators: this is the placeholder text for the main location entry.
+#: window.blp:188
+msgid "Search or enter a URL to open…"
+msgstr "Rechercher ou saisir une adresse…"
+
+#: Application.cpp:57
+msgid "SerenityOS developers"
+msgstr "Développeurs SerenityOS"
+
+#: Application.cpp:70
+msgid "SerenityOS website"
+msgstr "Site web de SerenityOS"
+
+#. Translators: Сергей Бугаев in Cyrillic
+#: Application.cpp:44
+msgid "Sergey Bugaev"
+msgstr "Sergey Bugaev"
+
+#: shortcuts-dialog.blp:8
+msgid "Shortcuts"
+msgstr "Raccourcis"
+
+#: shortcuts-dialog.blp:25
+msgid "Show shortcuts"
+msgstr "Afficher les raccourcis"
+
+#: window.blp:212
+msgid "Tab Overview"
+msgstr "Aperçu de l‘Onglet"
+
+#: shortcuts-dialog.blp:56
+msgid "Tabs"
+msgstr "Onglets"
+
+#. Translators: this is a menu item for unpinning a tab.
+#: window.blp:32
+msgid "Unp_in"
+msgstr "Désépingler"
+
+#: Tab.cpp:192
+msgid "Web page alerts:"
+msgstr "Message de la page:"
```

Comment:
It's not just a message, it's an alert, so more like a warning. (Though you can obviously use the API for any kind of message.) Is there not a more suitable word for this?

Reply:
I disliked the word `alert` for French (which comes from the API). Since what this API does is displaying a message, I used this one. Maybe these could be used:
- _Information_ de la page web
- _Indication_ de la page web

Reply:
_Indication_ sounds best to me among these, but this of course needs native speaker's (i.e. yours) judgement. _Message_ would also be fine, it's not a hard requirement not to use it 🙂